zilliqa/src/cfg.rs

```diff
@@ -159,6 +159,10 @@ pub struct NodeConfig {
     /// Maximum allowed RPC response size
     #[serde(default = "max_rpc_response_size_default")]
     pub max_rpc_response_size: u32,
+    /// The block height at which ZQ1 converted persistence ends. After this block ZQ2 blocks begin.
+    /// This value should be required only for proto networks to distinguise between ZQ1 and ZQ2 blocks. In future converted networks all ZQ1 blocks will be distinguishable by their zeroed state root hash.  
+    #[serde(default)]
+    pub proto_network_persistence_block_height: Option<u64>,
```

Comment:
Could it make sense to represent this property as a fork instead? i.e. we insert a fork at the first ZQ2 block with a new property that says "this block is executable" and all blocks before it aren't?

Reply:
Done. That's cleaner nice suggestion